lib/wallet/wallet_bitcoin.dart

```diff
@@ -5,12 +5,13 @@ import 'package:ten_ten_one/cfd_trading/cfd_trading.dart';
 import 'package:ten_ten_one/mocks/payment_history.dart';
 import 'package:ten_ten_one/models/service_model.dart';
 import 'package:ten_ten_one/wallet/payment_history_list_item.dart';
-import 'package:ten_ten_one/wallet/receive.dart';
 import 'package:ten_ten_one/wallet/seed.dart';
-import 'package:ten_ten_one/wallet/send.dart';
 import 'package:ten_ten_one/wallet/service_card.dart';
 import 'package:ten_ten_one/utilities/divider.dart';
 import 'package:flutter_speed_dial/flutter_speed_dial.dart';
+import 'package:ten_ten_one/wallet/withdraw.dart';
+
+import 'depsoit.dart';
```

Comment:
typo: dep**so**it.dart --> dep**os**it.dart

Also can you please change to a package import 🙂